code/drasil-utils/lib/Utils/Drasil/Console.hs

```diff
@@ -0,0 +1,13 @@
+module Utils.Drasil.Console (
+    -- * Colourized Console Messages
+    succMsg, warnMsg, errMsg
+) where
+
+succMsg :: String -> String
+succMsg msg = "\ESC[32m" ++ msg ++ "\ESC[0m"
+
+warnMsg :: String -> String
+warnMsg msg = "\ESC[33m" ++ msg ++ "\ESC[0m"
+
+errMsg :: String -> String
+errMsg msg = "\ESC[91m" ++ msg ++ "\ESC[0m"
```

Comment:
The console colour codes. Separate PR? Maybe use a separate Haskell package too?

Reply:
Should have been a separate PR. Would likely be best to leverage some existing Haskell package rather than maintaining this. Make an issue.

Reply:
Removed in 87efe7ef124357e9c0e5f7c2d576269a4ca7712d.